src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -392,6 +396,18 @@ export class ZaakViewComponent extends ActionsViewComponent implements OnInit, A
         if (this.zaak.heropend) {
             this.indicaties.push(new Indicatie('indicatieHeropend', this.zaak.status.toelichting));
         }
+        if (this.zaak.isDeelzaak) {
+            const hoofdzaak = this.zaak.gerelateerdeZaken.find(gerelateerdeZaak => gerelateerdeZaak.relatieType === ZaakRelatietype.HOOFDZAAK);
```

Comment:
Zet deze logica in een methode op de Zaak class: getHoofdzaak(). Ik weet echter niet of dit problemen gaat opleveren met JSON serializatie/deserialisatie. Aan de Java kant zetten we er dan een @JsionbTransient annotatie op. Ik weet niet hoe dit uitpakt in typescript. Kan zijn dat het gewoon goed gaat